python/lsst/ts/observing/utilities/auxtel/latiss/getters.py

```diff
@@ -30,8 +28,8 @@ async def get_image(
     dataset: `string`
         dataset to use when grabbing image.
         Examples include: 'raw','quickLookExp' (default)
-    datapath: `string`
-        Directory of butler repository
+    best_effort_isr: `object`
+        BestEffortISR class instantiated with a butler.
```

Comment:
Ah, so here you even say that's how you're doing it, further motivating the changes I mentioned in the other PR.